app/ui/browser-australis/sagas/external-sagas.js

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2016 Mozilla
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License. You may obtain a copy of the
+License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+*/
+
+import { takeLatest } from 'redux-saga';
```

Comment:
I don't see much value from separating sagas from actions in this context -- if I'm firing off an action from a container/view, how it gets processed shouldn't be of too much concern